packages/app-server-issuer/src/exchange-provider/config.ts

```diff
@@ -0,0 +1,13 @@
+import { BigNumber } from 'ethers';
```

Comment:
lumos has exposed a big integer library `@ckb-lumos/bi`, it is more light-weight, we can use it